geoportailv3/static/js/layermanager/layermanagerdirective.js

```diff
@@ -44,10 +45,24 @@ app.module.directive('appLayermanager', app.layermanagerDirective);
 
 /**
  * @constructor
+ * @param {angular.Scope} $scope
+ * @param {ngeo.SyncArrays} ngeoSyncArrays
  * @export
  */
-app.LayermanagerController = function() {
+app.LayermanagerController = function($scope, ngeoSyncArrays) {
   this['uid'] = goog.getUid(this);
+  ngeoSyncArrays(this['map'].getLayers().getArray(),
+      this['layers'], true, $scope,
+      goog.bind(function(layer) {
+        return goog.isDefAndNotNull(layer['values_']['metadata']);
```

Comment:
Testing that the layer is first in the list should be enough (ie. index == 0). This is what we do elsewhere.